src/components/bulk/config/ActionExportBulkOpConfig.scss

```diff
@@ -0,0 +1,16 @@
+.PaneBase {
```

Comment:
There is no need to wrap the SCSS in this file in a `.PaneBase` selector. In fact, the file itself is just top-level CSS, meaning that if anything, it should be wrapped in an `.ActionExportBulkOpConfig` selector.

However, because the next-level select, `.ActionExportBulkOpConfig-maxActionsWarning` is already specific enough to select the correct element, there is no need for any wrapping.

Conclusion: Remove the wrapping `.PaneBase {}` and unindent the entire content.